File: 10.Python/Swap.py
def main():
    No1=0
    No2=0
    temp=0
    print("Enter the one number\n");
    No1=int(input())

    print("Enter the second number\n");
    No2=int(input())
    
    print("before swap:No1={} No2={}\n".format(No1,No2))
    
    temp=No1
    No1=No2
    No2=temp

    print("After swap:No1={} No2={}".format(No1,No2))

if __name__ =="__main__":
    main()

File: 0.C_CPP_JAVA Programs/JAVA/Swap.java
import java.util.*;
import java.lang.*;
public class Swap
{
    public static void main(String[] args)
    {
        Scanner reader=new Scanner(System.in);
        System.out.println("Enter first number");
        int No1=reader.nextInt();

        System.out.println("Enter second number");
        int No2=reader.nextInt();

        System.out.format("Before swapping No1= %d  No2=%d ",No1,No2);
        int temp=No1;
        No1=No2;
        No2=temp;
        System.out.format("After swapping No1= %d  No2=%d ",No1,No2);
    }
}